### PR TITLE
HB dev requires Coq master

### PR DESCRIPTION
**Pull request by proux01**

### CODE REVIEW

**Review by gares — APPROVED** [review]

**Comment by gares** [comment]
```
  - install elpi                        1.14.3  [required by coq-elpi]
  - install coq-elpi                    dev     [required by coq-hierarchy-builder]
  - install coq-hierarchy-builder       dev
 ```
This is weird, since coq-elpi#coq-master should use elpi 1.15.2

**Comment by proux01** [comment]
This is just the bound in coq-elpi.dev that's not up to date, let's fix that too.

**Comment by proux01** [comment]
Now the test-suite fails with `wdiff: not found`

File: extra-dev/packages/coq-hierarchy-builder/coq-hierarchy-builder.dev/opam [hunk 1]
```diff
@@ -10,7 +10,10 @@ build: [ [ make "build"]
          [ make "test-suite" ] {with-test}
        ]
 install: [ make "install" ]
-depends: [ "coq-elpi" {>= "1.11.0"} ]
+depends: [ "coq-elpi" {>= "1.14"} "coq" {= "dev"} ]
+depexts: [
+  "wdiff" {os-family = "debian" & with-test}
```
Comment (silene) [inline extra-dev/packages/coq-hierarchy-builder/coq-hierarchy-builder.dev/opam:15]:
```suggestion
  [ "wdiff" ] {os-family = "debian" & with-test}
```

**Pull request merged**